docs/source/quickstart.rst

```diff
@@ -91,187 +113,206 @@ as follows (this example is also included in ``quickstart/mlflow_tracking.py``):
+
+For example:
+
+.. code-section::
 
     .. code-block:: python
 
         import mlflow
 
-        mlflow.set_tracking_uri("http://YOUR-SERVER:4040")
-        mlflow.set_experiment("my-experiment")
+        from sklearn.model_selection import train_test_split
+        from sklearn.datasets import load_diabetes
+        from sklearn.ensemble import RandomForestRegressor
+
+        with mlflow.start_run() as run:
+            # Load the diabetes dataset.
+            db = load_diabetes()
+            X_train, X_test, y_train, y_test = train_test_split(db.data, db.target)
 
-    .. code-block:: R
+            # Create and train models.
+            rf = RandomForestRegressor(n_estimators=100, max_depth=6, max_features=3)
+            rf.fit(X_train, y_train)
 
-        library(mlflow)
-        install_mlflow()
-        mlflow_set_tracking_uri("http://YOUR-SERVER:4040")
-        mlflow_set_experiment("/my-experiment")
+            # Use the model to make predictions on the test dataset.
+            predictions = rf.predict(X_test)
+            print(predictions)
 
+            mlflow.sklearn.log_model(rf, "model")
 
-Log to Databricks Community Edition
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+            print("Run ID: {}".format(run.info.run_id))
 
-Alternatively, sign up for `Databricks Community Edition <https://databricks.com/try-databricks>`_,
-a free service that includes a hosted tracking server. Note that
-Community Edition is intended for quick experimentation rather than production use cases.
-After signing up, run ``databricks configure`` to create a credentials file for MLflow, specifying
-https://community.cloud.databricks.com as the host.
+In the case of the ``sklearn`` flavor, ``log_model`` stores the following files in the **artifacts** directory of the run's directory on the tracking server:
 
-To log to the Community Edition server, set the ``MLFLOW_TRACKING_URI`` environment variable
-to "databricks", or add the following to the start of your program:
+.. code-section::
+
+    .. code-block:: shell
 
-  .. code-section::
+        model/
+        |-- MLmodel
+        |-- conda.yaml
+        |-- model.pkl
+        |-- python_env.yaml
+        |-- requirements.txt
+
+If you've not called ``set_tracking_uri`` or set the ``MLFLOW_TRACKING_URI`` environment variable to point to a remote tracking server, this **model** directory will be under the **mlruns** directory.
+
+For more information, including a list of supported model flavors and storing your own flavor, see :ref:`models_built-in-model-flavors`.
+
+Load a model from a specific training run for inference
+--------------------------------------------
```

Comment:
```suggestion
Load a model from a specific training run for inference
------------------------------------------------------
```
.rst linting is complaining about this. We have lint checks enabled that mandate the section delimiter must be at least as long as the title.